### DISCUSSION TITLE
Dude, there's a bug in the code

### DISCUSSION THREAD

**Discussion by daidaiJ** [PaddlePaddle/PaddleNLP#274] [Q&A] [answered]
when i run the 'erniecrffortokenclassification.py',There was a strange bug,
like this ;'TypeError: forward() missing 1 required positional argument: 'labels''
This error is happend in Fluid DyGraph's Layers.py
But I obviously passed in labels, otherwise the error would be in the line 65
     64         if labels is not None:
---> 65             loss = self.crf_loss(logits, lengths, labels)
     66             return loss
     67         else:
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/paddle/fluid/dygraph/layers.py in __call__(self, *inputs, **kwargs)
    900                 self._built = True
    901 
--> 902             outputs = self.forward(*inputs, **kwargs)
    903 
    904             for forward_post_hook in self._forward_post_hooks.values():
TypeError: forward() missing 1 required positional argument: 'labels'

**Comment by ZeyuChen** [comment, marked as answer]
Could you provide your paddlenlp version and paddlepaddle version?
Please make sure you are using the latest version.

**Reply by ZeyuChen** [reply]
@daidaiJ

**Reply by daidaiJ** [reply]
Em, I checked that the version of the module is 2.0.0rc7, is it because the version does not match?

**Reply by ZeyuChen** [reply]
Maybe @daidaiJ Please upgrade to the latest version rc18 and try again.

**Reply by daidaiJ** [reply]
thanks , ahhhh~
